### ISSUE TITLE
SASS Beautify strips "ms" from "0" values

### ISSUE THREAD

**Issue by brandonmerritt** [Glavin001/atom-beautify#1909]
# Description
On save Beautify strips the "ms/s/etc" values from any "0" on save. Various browsers (specifically Chrome) used to allow the time units to be neglected in previous versions but that bug is now fixed. All browsers require a time unit (even 0).

https://stackoverflow.com/questions/34966221/why-does-this-css-transition-shorthand-not-work-anymore 

# Input Before Beautification

This is what the code looked like before:

```
  -webkit-transition: left 400ms ease-in-out 0ms;
  -moz-transition: left 400ms ease-in-out 0ms;
  -o-transition: left 400ms ease-in-out 0ms;
  transition: left 400ms ease-in-out 0ms;
```

# Expected Output

The beautified code should have looked like this:

```
  -webkit-transition: left 400ms ease-in-out 0ms;
  -moz-transition: left 400ms ease-in-out 0ms;
  -o-transition: left 400ms ease-in-out 0ms;
  transition: left 400ms ease-in-out 0ms;
```
OR
```
  -webkit-transition: left 400ms ease-in-out;
  -moz-transition: left 400ms ease-in-out;
  -o-transition: left 400ms ease-in-out;
  transition: left 400ms ease-in-out;
```
# Actual Output

The beautified code actually looked like this:

```
  -webkit-transition: left 400ms ease-in-out 0;
  -moz-transition: left 400ms ease-in-out 0;
  -o-transition: left 400ms ease-in-out 0;
  transition: left 400ms ease-in-out 0;
```
# Steps to Reproduce

1. Add code to Atom editor
```
.some-transition {
    -webkit-transition: box-shadow 400ms ease-in-out 0ms;
    -moz-transition: box-shadow 400ms ease-in-out 0ms;
    -o-transition: box-shadow 400ms ease-in-out 0ms;
    transition: box-shadow 400ms ease-in-out 0ms;
  }
```
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug
Not actually a "failure" so no debug info.

# Checklist

I have:
- [x ] Installed the latest version of Beautify
- [ x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [ x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues so I know this is not a duplicate issue
- [ x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ ] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by szeck87** [comment]
This sounds like a says issue. Have you opened an issue with them?

**Comment by brandonmerritt** [comment]
It definitely happens only when beautifying SASS. If you can point me to the appropriate project I'm happy to open an issue there. (Sorry for the ignorance, but I'm unsure if you are including a SASS specific package or some 3rd party.)

**Comment by szeck87** [comment]
Actually, can you confirm it's SASS (sass-convert) you're using to beautify?  If you run the Debug Atom Beautify command against the file, it will show you the **Selected Beautifier**: towards the top of the contents of the generated file.

**Comment by brandonmerritt** [comment]
This is what i'm getting:
```
**Supported Beautifiers**: CSScomb, Pretty Diff, SassConvert
**Selected Beautifier**: Pretty Diff
```
Interestingly, my file has an .scss extension vs .sass so I'm assuming "Pretty Diff" is the issue. If I change the setting for SCSS to SassConvert I don't run in to the same issue.

<img width="907" alt="screen shot 2017-10-25 at 7 47 47 pm" src="https://user-images.githubusercontent.com/813414/32028675-ceeb3820-b9bd-11e7-9a23-7495b60866f5.png">

**Comment by szeck87** [comment]
Yep, looks like it.  @prettydiff it looks similar to https://github.com/prettydiff/prettydiff/issues/463, which was fixed in 2.2.8 and hasn't been published to npm yet for Atom Beautify to use.

But, I also tried on prettydiff.com which has 2.2.9 and it still stripped the `ms` after ease-in-out.

```
.some-transition {
    -webkit-transition: box-shadow 400ms ease-in-out 0;
    -moz-transition: box-shadow 400ms ease-in-out 0;
    -o-transition: box-shadow 400ms ease-in-out 0;
    transition: box-shadow 400ms ease-in-out 0ms;
}
```

**Comment by prettydiff** [comment]
I can reproduce the defect in Pretty Diff.  I am opening a defect there to work on this.
